Sources/Auth/Internal/SessionManager.swift

```diff
@@ -40,7 +40,7 @@ private actor LiveSessionManager {
   func session() async throws -> Session {
     try await trace(using: logger) {
       guard let currentSession = try sessionStorage.get() else {
-        throw AuthError.sessionNotFound
+        throw AuthError.sessionMissing
```

Comment:
I feel like this could technically be considered a breaking change, but I will leave it up to you what to do with it.

Reply:
Not really since I added a `static let sessionNotFound: AuthError { .sessionMissing }`, and deprecated it.

In a Swift enum, `cases` and `static let` are "kinda" equivalent.